src/SWP/Component/OutputChannel/Provider/WordpressAdapterProvider.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Superdesk Web Publisher Output Channel Component.
+ *
+ * Copyright 2018 Sourcefabric z.ú. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2018 Sourcefabric z.ú
+ * @license http://www.superdesk.org/license
+ */
+
+namespace SWP\Component\OutputChannel\Provider;
+
+use SWP\Component\OutputChannel\Adapter\AdapterInterface;
+use SWP\Component\OutputChannel\Model\OutputChannelInterface;
+
+final class WordpressAdapterProvider implements AdapterProviderInterface
```

Comment:
For what we need it? Shouldn't `AdapterProviderChain` already give us adapter by checking if it supports provided output?